connectors/meilisearch/index.js

```diff
@@ -112,12 +112,13 @@ module.exports = async ({ storeConnector, collectionConnector }) => {
     /**
      * Get indexes with a safe guard in case of error.
      *
-     * @returns { string[] }
+     * @returns { Promise<Record<string, any>[]> }
      */
     getIndexes: async function () {
       try {
         const client = MeiliSearch({ apiKey, host })
-        return await client.getIndexes()
+        const indexes = await client.getIndexes()
+        return indexes.results || []
```

Comment:
I don't think the `|| []` is necessary here because if, for some reason, the `getIndexes` thrown an exception it will be retrieved by the `try/catch`, otherwise the `indexes` var will contain the correct format and at least an array, right?